docker-compose.yml

```diff
@@ -59,9 +68,21 @@ services:
     build: *web_build
     environment: *web_env
     command: /celery_worker.sh
+    healthcheck:
+      test: ["CMD", "python", "/app/bin/check_celery_worker_liveness.py"]
+      interval: 30s
+      timeout: 5s
+      retries: 3
+      start_period: 10s    
     depends_on:
-      - db
-      - redis
+      web:
+        # This health check condition is needed because Celery Beat will 
```

Comment:
We don't have celery beat for Objects API, so I think this comment is not needed

Reply:
Oke, I dont even think the comment is correct.